inc/user.class.php

```diff
@@ -484,11 +484,11 @@ function getFromDBbyEmail($email, $condition = []) {
                   ]
                ]
             ],
-            'WHERE'     => ['glpi_useremails.email' => $email] + $condition
+            'WHERE'     => ['glpi_useremails.email' => $email]+$condition
```

Comment:
this one may be bad

Reply:
indeed... some proposal are interesting, others are maybe not :/